### DISCUSSION TITLE
⏳ 🔗 New Border Radius Control

### DISCUSSION THREAD

**Discussion by Henzxon** [elementor/elementor#22612] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The custom border radius uses "TOP" "RIGHT" "BOTTOM" "LEFT" I think this is incorrect. While most users or designer know that the order is clockwise from top. Sometimes this is confusing since the "top" label is for sides and not for corners.
<img width="304" alt="image" src="https://github.com/elementor/elementor/assets/102000469/ffff51ca-8414-4721-926d-c1eec5a87707">

### Describe the solution you'd like

I think a better corner indicator should be implemented just like photoshop?
<img width="211" alt="image" src="https://github.com/elementor/elementor/assets/102000469/164e55b6-7253-46e9-a5c5-17d1941a9306">

or like how figma does it
<img width="267" alt="image" src="https://github.com/elementor/elementor/assets/102000469/95a89db6-4b0f-4008-b852-649f2fe26ebd">

Concept
<img width="324" alt="image" src="https://github.com/elementor/elementor/assets/102000469/8f4da077-7816-4cea-9f6a-1a7698f85c05">

### Describe alternatives you've considered

N/A

**Comment by rami-elementor** [comment]
Hi @Henzxon,

Thank you for your detailed suggestion.

We have actually been researching the `border-radius` control for quite a while.

I love the concept you designed. However, our new border-radius control will provide even more options for designers and web-creators.

Can’t reveal more information, but rest assured that the new control will offer features that no other platform currently supports.

Rami Y.
Elementor.com

**Reply by lensbreak** [reply]
Unless Elementor takes 6-12 months to release that new ground-breaking radius features. You posted in September 2023. Whats the status now? :-)
